app/models/uploaded_evidence_collection.rb

```diff
@@ -26,8 +26,9 @@ def all_mandatory_evidence_uploaded
     mandatory_evidence_types.each do |type|
       next if categorised_evidence_types.include?(type)
 
+      matter_type = legal_aid_application.public_law_family_proceedings? ? "" : "for Section 8 proceedings"
       # link the error message to the dropzone
-      errors.add("dz-upload-button", I18n.t("#{error_path}.#{type}_missing", benefit: passporting_benefit_title), mandatory_evidence: true)
+      errors.add("dz-upload-button", I18n.t("#{error_path}.#{type}_missing", benefit: passporting_benefit_title, matter_type:), mandatory_evidence: true)
```

Comment:
nit: could call the translations argument `suffix` since it is just language rather than being murely a matter_type but am of two minds myself so either way.